src/corelibs/U2View/src/ov_phyltree/TreeViewer.cpp

```diff
@@ -1407,6 +1409,12 @@ void TreeViewerUI::sl_showDistanceLabelsTriggered(bool on) {
 }
 
 void TreeViewerUI::sl_printTriggered() {
+#ifdef Q_OS_WIN
```

Comment:
Use isOsWindows() instead of the macro. It has 2 benefits:
1. Checks on compile time all branches 
2. Checks that isOsWindows function is correctly included: macro may be not included at all.